views/12-months.html

```diff
@@ -0,0 +1,39 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+You cannot use this service
+{% endblock %}
+{% block beforeContent %}
+{{ govukPhaseBanner({
+tag: {
+text: "alpha"
+},
+html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+  }) }}
+{{ govukBackLink({
```

Comment:
Not sure if the backlink is needed, it's not on the screenshot in the story. if it is needed, it should be implemented same as other extensions backlinks as extensions is using a stack to trace the page history and this could possibly bypass this without updating the stack.

Reply:
Removed